### PR TITLE
Support auto imports in highlight

### PR DESCRIPTION
**Pull request by PolinB**

### CODE REVIEW

File: src/webdemo-api.js [hunk 1]
```diff
@@ -130,14 +131,14 @@ export default class WebDemoApi {
    * Request for getting errors of current file
    *
    * @param code - string code
-   * @param compilerVersion - string kotlin compiler
+   * @param compißlerVersion - string kotlin compiler
```
Comment (AlexanderPrendota) [inline src/webdemo-api.js:134]: compi**ß**lerVersion?

File: src/executable-code/executable-fragment.js [hunk 2]
```diff
@@ -330,6 +335,19 @@ export default class ExecutableFragment extends ExecutableCodeTemplate {
     }
   }
 
+  prefixEmptyOrContainsOnlyImports() {
+    if (!this.prefix) return true
+    let prefix = this.prefix
+    let textLines = prefix.split("\n");
+    for(let i = textLines.length - 1; i >= 0; --i) {
+      let line = textLines[i]
```
Comment (zoobestik) [inline src/executable-code/executable-fragment.js:343]: Looks difficult to read, may we use collection methods:
 ```javascript
prefixEmptyOrContainsOnlyImports(prefix) {
    return prefix.split("\n").every(line => ...);
}
```

File: src/executable-code/executable-fragment.js [hunk 3]
```diff
@@ -555,9 +588,48 @@ export default class ExecutableFragment extends ExecutableCodeTemplate {
...
+        let withImports = this.canAddImport;
+        if (results.length !== 0) {
+          let options = {
+            hint: function () {
+              return {
+                from: mirror.getDoc().getCursor(),
+                to: mirror.getDoc().getCursor(),
+                list: results.map(result => {
+                  if (!withImports) {
+                    result.import = undefined
```
Comment (zoobestik) [inline src/executable-code/executable-fragment.js:617]: I am not sure, It is so clear for me. Can you explain more?..

**Review by zoobestik — COMMENTED** [review]
@PolinB can you provide test cases for feature in PR description?

File: src/executable-code/executable-fragment.js (hunk 3 above)
Comment (PolinB) [reply src/executable-code/executable-fragment.js:617]: I had a problem with the addition of imports, if there is a hidden code at the top. So I set the flag, depending on the header, and then, if I can't add it, I ignore the passed imports.

File: src/executable-code/executable-fragment.js [hunk 4]
```diff
@@ -363,12 +381,21 @@ export default class ExecutableFragment extends ExecutableCodeTemplate {
 
       const errorMessage = unEscapeString(diagnostic.message);
       const severity = diagnostic.severity;
+      const containsSuggestions = diagnostic.imports !== undefined;
```
Comment (AlexanderPrendota) [inline src/executable-code/executable-fragment.js:384]: !!diagnostic.imports, I'm right?

**Review by AlexanderPrendota — CHANGES_REQUESTED** [review]

**Pull request merged**